### PR TITLE
Add documentation for data-turbo-preload

### PR DESCRIPTION
**Pull request by hey-leon**
This PR adds the documentation for https://github.com/hotwired/turbo/pull/552. Feedback on the documentation is of course welcome!

It appears the plan for the release of the feature is in 7.2 - https://github.com/hotwired/turbo/pull/552#issuecomment-1160493227. I could not see any 7.2 so I am not positive what would be the correct base branch to use here, so in lieu I have assigned main branch.

Here is a screenshot of the changes in development.

image
image

### CODE REVIEW

File: _source/handbook/02_drive.md [hunk 1]
```diff
@@ -274,4 +274,12 @@ If the form submission is a GET request, you may render the directly rendered re
 ## Streaming After a Form Submission
 
 Servers may also respond to form submissions with a [Turbo Streams](streams) message by sending the header `Content-Type: text/vnd.turbo-stream.html` followed by one or more `<turbo-stream>` elements in the response body. This lets you update multiple parts of the page without navigating.
+
+## Preload links into turbo's cache
```
Comment (ghiculescu) [inline _source/handbook/02_drive.md:278]:
```suggestion
## Preload Links Into the Cache
```

All other headers have most words capitalised, so I think we should match that.

File: _source/handbook/02_drive.md [hunk 2]
```diff
@@ -274,4 +274,12 @@ If the form submission is a GET request, you may render the directly rendered re
...
+
+Preload links into turbo's cache using `<a href="/" data-turbo-preload>Home</a>`.
```
Comment (ghiculescu) [inline _source/handbook/02_drive.md:280]:
```suggestion
Preload links into Turbo Drive's cache using `<a href="/" data-turbo-preload>Home</a>`.
```

Comment (hey-leon) [reply _source/handbook/02_drive.md:280]: Good call 👍

**Comment by hey-leon** [comment]
Looks like CI failure is simply from a rate limit being exceeded.

**Pull request merged**